src/lib/pci_virtio_sock.c

```diff
@@ -2008,6 +2012,7 @@ static ssize_t pci_vtsock_proc_rx(struct pci_vtsock_softc *sc,
 		dprint_header(hdr, 0, "RX");
 		s->credit_update_required = false;
 		vq_relchain(vq, idx, sizeof(*hdr));
+		s->credit_cnt = hdr->fwd_cnt;
```

Comment:
The four places with this `s->credit_update_required = false; s->credit_cnt = hdr->fwd_cnt;` pattern could profitably become `clear_credit_update_required` (cf `set_credit_update_required`) perhaps, which would be somewhat self documenting through the naming.